fenix/app/src/debug/AndroidManifest.xml

```diff
@@ -26,6 +26,8 @@
     <application
             tools:replace="android:name"
             android:name="org.mozilla.fenix.DebugFenixApplication">
+
+        <activity android:name="androidx.activity.ComponentActivity" />
```

Comment:
for some reason, and only in the Fenix manifest, I needed to add this activity tag in order to get `createComposeRule()` to work in a UI test in Fenix. (An empty `ComponentActivity` is stood up via `createComposeRule()` that we can then set the content on. This way we don't have to do something as costly as working around `HomeActivity` to just test a Modifier.